lib/fxpay/utils.js

```diff
@@ -197,4 +197,20 @@ define([
         versionDeprecated);
   };
 
+ /**
+  * Take an object of key value pairs and serialize it
```

Comment:
maybe elaborate that 'serialize' means to turn it into a URL-encoded query string.